packages/bot-web-ui/src/components/transactions/transactions.tsx

```diff
@@ -12,9 +11,12 @@ import { transaction_elements } from 'Constants/transactions';
 import { useDBotStore } from 'Stores/useDBotStore';
 import Transaction from './transaction.jsx';
 
+type TTransactions = {
+    is_drawer_open: boolean;
+};
+
 const TransactionItem = ({ row, is_new_row }) => {
```

Comment:
It's not relevant but can we also add type to the `TransactionItem` component? @farabi-deriv

Reply:
Sure. Added @sandeep-deriv